app/frontend/vite.config.ts

```diff
@@ -20,7 +20,8 @@ export default defineConfig({
                     }
                 }
             }
-        }
+        },
+        target: "esnext"
```

Comment:
I'm curious why this change was needed? (Using more recent syntax?)

Reply:
yes, top level await. I actually believe the intent was to use esnext because it's present in the node files